domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -376,10 +422,27 @@ void setClient(PgPool client) {
   }
 
   /**
-   * Close the SQL client of this PostgresClient instance.
+   * Close the SQL writer and reader clients of this PostgresClient instance in succession.
    * This is idempotent: additional close invocations are always successful.
    */
-  public Future<Void> closeClient() {
+  public Future<Void> closeReadClient() {
```

Comment:
closeReadClient is unused. We should not add useless code. So I suggest that to be removed.

Reply:
Yes, it can be removed. I was trying various ways to get it to work and this may be a residue from doing it.

Reply:
OK. Will remove this and other one.